toolkit/tools/imager/imager.go

```diff
@@ -600,6 +605,9 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 		return
 	}
 
+	// Copy image package manifest from 'installroot' chroot to 'setuproot' chroot
+	installChroot.ExtractFile("image_pkg_manifest_temp1.json", "/image_pkg_manifest_temp2.json")
```

Comment:
nit: maybe have more meaningful names than temp1, temp2. Temp sounds like something you were testing with, and doesn't tell me what the file's purpose is